src/runtimes/runtime.ts

```diff
@@ -49,9 +57,18 @@ type ZipFunction = (
 
 interface Runtime {
   findFunctionsInPaths: FindFunctionsInPathsFunction
+  getFunctionAtPath: GetFunctionAtPathFunction
   getSrcFiles?: GetSrcFilesFunction
   name: RuntimeName
   zipFunction: ZipFunction
 }
 
-export { FindFunctionsInPathsFunction, GetSrcFilesFunction, Runtime, RuntimeName, ZipFunction, ZipFunctionResult }
+export {
+  FindFunctionsInPathsFunction,
+  GetSrcFilesFunction,
+  Runtime,
+  RuntimeName,
+  ZipFunction,
+  ZipFunctionResult,
+  GetFunctionAtPathFunction,
```

Comment:
[dust] Can we maintain alphabetical order of properties?

Reply:
outdated by https://github.com/netlify/zip-it-and-ship-it/pull/899/commits/15c1cadfe33854a8bee6810cd66923f22f22a089